### DISCUSSION TITLE
Internal action searchable through marketplace for Enterprise accounts

### DISCUSSION THREAD

**Discussion by anirudhkoutha-o** [community/community#15699] [Enterprise]
With Feature #254, we are able to refer to internal actions across Organizations in an Enterprise, but they are not available/searchable through marketplace.

This feature would provide developers with the ability to search across the wider actions available. Can you make this feature available?

**Comment by ordovicia** [comment]
I would also be very grateful if this feature becomes available!
I want to share my Actions within our Enterprise Cloud and make the Actions discoverable in a closed Marketplace for the enterprise.
